lms/djangoapps/bulk_email/tests/test_models.py

```diff
@@ -25,9 +25,9 @@ def test_creation(self):
         to_option = SEND_TO_STAFF
         subject = "dummy subject"
         html_message = "<html>dummy message</html>"
-        email = CourseEmail.create(course_id, sender, to_option, subject, html_message)
+        email = CourseEmail.create(course_id, sender, [to_option], subject, html_message)
         self.assertEquals(email.course_id, course_id)
-        self.assertEquals(email.to_option, SEND_TO_STAFF)
+        self.assertEquals(email.targets.all()[0].target_type, SEND_TO_STAFF)
```

Comment:
If it's not relevant for other functionality that `SEND_TO_STAFF` always come first in this list, consider changing this to an `assertTrue` that `targets` contains `SEND_TO_STAFF`.